From b6bfece0d9ddf21e1526fead81340ef02f98f6ad Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.com>
Date: Tue, 17 Oct 2023 17:28:11 +0100
Subject: [PATCH] dts: bcm2712: Use the new model name

"Model B" is no more - "Raspberry Pi 5" is the official name.

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 arch/arm/boot/dts/bcm2712-rpi-5-b.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/arch/arm/boot/dts/bcm2712-rpi-5-b.dts
+++ b/arch/arm/boot/dts/bcm2712-rpi-5-b.dts
@@ -52,7 +52,7 @@
 
 / {
 	compatible = "raspberrypi,5-model-b", "brcm,bcm2712";
-	model = "Raspberry Pi 5 Model B";
+	model = "Raspberry Pi 5";
 
 	/* Will be filled by the bootloader */
 	memory@0 {
